kyuubi-server/src/main/scala/org/apache/kyuubi/server/metadata/api/Metadata.scala

```diff
@@ -66,6 +68,8 @@ case class Metadata(
     createTime: Long = 0L,
     engineType: String = null,
     clusterManager: Option[String] = None,
+    kubernetesContext: Option[String] = None,
+    kubernetesNamespace: Option[String] = None,
```

Comment:
will this cause metadata DDL change? I prefer to store them in requestConf, and make them as method instead of `val`.

Reply:
Good idea